test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+	private Utils utils;
+	
+	@Before
+	public void setup() {
+		utils = new Utils();
```

Comment:
It is good to use `@Before` for setting up each test case. But note that in our program, `Utils` only has static methods, so there is no need to create a new object for each test case.